extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -510,22 +524,47 @@ export class IntergrationRuntimePage extends MigrationWizardPage {
 				CSSStyles: { ...styles.BODY_CSS }
 			}).component();
 
+		const refreshAndConfigureIRcontainer = this._view.modelBuilder.toolbarContainer()
+			.withToolbarItems(
+				[
+					{ component: this._refreshButton },
+					{ component: _configureIRButton }
+				]
+			).component();
+
+		this._integrationRuntimeTable = utils.createIntegrationRuntimeTable(this._view);
+
+		/*const statusContainer = this._view.modelBuilder.flexContainer()
+			.withLayout({ flexFlow: 'column' })
```

Comment:
can this commented out code be removed?  what is it for?

Reply:
Removed